pinot-core/src/main/java/org/apache/pinot/core/io/writer/impl/BaseChunkSVForwardIndexWriter.java

```diff
@@ -27,21 +27,20 @@
 import java.nio.channels.FileChannel;
 import org.apache.pinot.core.io.compression.ChunkCompressor;
 import org.apache.pinot.core.io.compression.ChunkCompressorFactory;
-import org.apache.pinot.core.io.writer.SingleColumnSingleValueWriter;
+import org.apache.pinot.core.io.writer.ForwardIndexWriter;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 
 /**
- * Abstract implementation for {@link SingleColumnSingleValueWriter}
- * Base class for fixed and variable byte writer implementations.
+ * Base implementation for chunk based single-value forward index writer.
```

Comment:
Consider adding "raw"  -- `Base implementation for chunk based single-value raw forward index writer.`